templates/scenarios/ts/link-unfurling/card.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Are we getting rid of `src` folder? Also can we follow the pattern for notification/command bot where the adaptive cards are put under `/adaptiveCards` path and probably name it as `helloworldLinkUnfurlingCard.json` or something similar.

Reply:
Maybe helloWorldCard.json is enough?